submissions/franchukv/js-oop-frogger/app.js

```diff
@@ -0,0 +1,136 @@
+const CANVAS_HEIGHT = 606;
+const CANVAS_WIDTH = 505;
+const TILE_HEIGHT = 83;
+const TILE_WIDTH = 101;
+
+const PLAYER_CONF = {
+    size: {
+        height: 40,
+        width: 40
+    },
+    position: {
+        x: 202,
+        y: 380
+    }
+};
+
+const ENEMIES_CONF = {
+    size: {
+        height: 40,
+        width: 80
+    },
+    enemies: [{
+        x: -80,
```

Comment:
You could pass repeating value at `new Enemy` moment.
E.g. `new Enemy({ x: -80, ...ENEMY_CONF}, ....)`
Would make enemies config 33% shorter. This would also help to change the number in a single place should we need to do this.